mslib/mscolab/models.py

```diff
@@ -42,7 +42,9 @@ class User(db.Model):
     password = db.Column(db.String(255), unique=True)
     registered_on = db.Column(db.DateTime, nullable=False)
     confirmed = db.Column(db.Boolean, nullable=False, default=False)
-    confirmed_on = db.Column(db.DateTime, nullable=True)
+    confirmed_on = db.Column(db.DateTime(timezone=True),
+                             nullable=True,
+                             default=datetime.datetime.now(tz=datetime.timezone.utc))
```

Comment:
This shouldn't have a default: when creating a user their mail address is usually not confirmed immediately at creation, but at a later point in time. (Or the default should be None, which I think it is with `nullable=True` but no default set).